app/controllers/reviews_controller.rb

```diff
@@ -19,6 +19,27 @@ def create
     end
   end
 
+  def edit
+    @spot = Spot.find params[:spot_id]
+    @review = Review.find(params[:id])
+    @ratings = [1, 2, 3, 4, 5]
+  end
+
+  def update
+    @spot = Spot.find params[:spot_id]
+    @review = Review.find(params[:id])
+    @review.update(review_params)
```

Comment:
There should be an `if` statement that checks to see if the review was successfully updated or not, and flashes a relevant error message